.travis.yml

```diff
@@ -1,5 +1,3 @@
 language: rust
 rust:
-    - stable
```

Comment:
Because of the use of `impl Iterator` we can use only nightly. However, we should create a `FileNames` struct and return that instead.